ironfish-cli/src/commands/service/faucet.ts

```diff
@@ -126,14 +126,27 @@ export default class Faucet extends IronfishCommand {
       return
     }
 
-    let faucetTransactions = await api.getNextFaucetTransactions(MAX_RECIPIENTS_PER_TRANSACTION)
+    const baseFaucetTransactions = await api.getNextFaucetTransactions(
```

Comment:
nit: `base` seems a little confusing to me. I think leaving this as `faucetTransactions` (or `unprocessedFaucetTransactions`) and using `validFaucetTransactions` and `invalidFaucetTransactions` below would be a little clearer.